src/js/utils/NavigationServiceUtil.js

```diff
@@ -40,13 +40,19 @@ const NavigationServiceUtil = {
           route.component.routeConfig.label,
           { category, icon }
         );
-
         childRoutes
           .filter(({isInSidebar}) => isInSidebar)
           .forEach((childRoute) => {
+            // TODO: Temporary fix to allow routes to register under a different
```

Comment:
Because this function has been deprecated. When we make the effort to remove this function (and start registering routes using the `NavigationService` directly) we should also remove the added sidebarPath that was added to the services routes in this PR. This `TODO` is just to remind whoever will take this on to also do that